Source/WebKit/UIProcess/API/Cocoa/_WKAutomationSessionConfiguration.h

```diff
@@ -35,6 +35,7 @@ WK_CLASS_AVAILABLE(macos(10.13.4), ios(12.2))
 @property (nonatomic) BOOL acceptInsecureCertificates;
 @property (nonatomic) BOOL allowsInsecureMediaCapture;
 @property (nonatomic) BOOL suppressesICECandidateFiltering;
+@property (nonatomic) BOOL alwaysAllowAutoplay;
```

Comment:
what does this ultimately do?  should it be used to set something like `AutoplayPolicy` or `InvisibleAutoplayNotPermitted` before the page is created?  or perhaps passed down to eventually be used in `MediaElementSession::autoplayPermitted`?

Reply:
For users of the WebKit API which implement the (admittedly currently private) [`handleAutoplayEvent`](https://github.com/WebKit/WebKit/blob/e7c2bdd9fbcb321371c484bfeed5b1933253ba7b/Source/WebKit/UIProcess/API/Cocoa/WKUIDelegatePrivate.h#L197) delegate, the decision whether or not to autoplay is owned by the embedder.